.github/workflows/lintdiff-code.yaml

```diff
@@ -68,8 +51,7 @@ jobs:
           npm exec --no -- lint-diff \
             --before before \
             --after after \
-            --changed-files-path changed-files.txt \
-            --base-branch ${{ github.event.pull_request.base.ref }} \
+            --base-branch ${{ github.event.pull_request.base.sha }} \
```

Comment:
This makes a small change to the output markdown. Instead of linking to the base branch in the summary, the **base version** link points to the base branch's head SHA at the time of LintDiff's execution: 

<img width="690" height="226" alt="image" src="https://github.com/user-attachments/assets/3d08e35c-8e30-4dc1-8b14-6a6afa54b2d9" />


It's more accurate and slightly less readable. 

Alternatively, we could fetch refs in the `after` folder but that would make checkout take significantly longer.